src/interfaces/IMetaMorpho.sol

```diff
@@ -159,6 +159,7 @@ interface IMetaMorphoBase {
     /// market to be empty can be griefed by a front-run. To circumvent this, the allocator can simply bundle a
     /// reallocation that withdraws max from this market with a call to `updateWithdrawQueue`.
     /// @dev Warning: Removing a market with supply will decrease the fee accrued until the next deposit/withdrawal.
+    /// @dev Warning: updateWithdrawQueue is not idempotent. Submitting twice the same tx will change the queue twice.
```

Comment:
Nitpick: updateWithdrawQueue here is not wrapped by ` but in the above comments it is.

Reply:
nitpicking goes brrrrr

Reply:
> nitpicking goes brrrrr

it's a trap!